### ISSUE TITLE
/entity/dependency does not output final newline

### ISSUE THREAD

**Issue by hlapp** [phenoscape/phenoscape-kb-services#130]
This leads to the following warning in R:
```R
read.csv(url('https://kb.phenoscape.org/api/entity/dependency?terms=["http://purl.obolibrary.org/obo/UBERON_0000981","http://purl.obolibrary.org/obo/UBERON_0002103","http://purl.obolibrary.org/obo/UBERON_0000976","http://purl.obolibrary.org/obo/UBERON_0002102"]'))
```
```
Warning message:
In read.table(file = file, header = header, sep = sep, quote = quote,  :
incomplete final line found by readTableHeader on 'https://kb.phenoscape.org/api/entity/dependency?terms=["http://purl.obolibrary.org/obo/UBERON_0000981","http://purl.obolibrary.org/obo/UBERON_0002103","http://purl.obolibrary.org/obo/UBERON_0000976","http://purl.obolibrary.org/obo/UBERON_0002102"]'
```
I can turn the output into a string and then use `textConnection(content)` instead of `url()`, but other clients may not be made tolerant as easily, so worth fixing.

**Comment by balhoff** [comment]
@Shalsh23 can you update this today?

**Comment by balhoff** [comment]
Fixed by c8fe3213acc54ecea764bbd2d7cc11e090aed682.
